tapeagents/llms.py

```diff
@@ -91,12 +91,18 @@ def make_training_text(self, prompt: Prompt, output: LLMOutput) -> TrainingText:
         pass
 
     def log_output(self, prompt: Prompt, message: LLMOutput, cached: bool = False):
+        prompt_length_tokens = self.count_tokens(prompt.messages)
+        if message.content:
+            output_length_tokens = self.count_tokens(prompt.messages + [{"role": "assistant", "content": message.content}]) - prompt_length_tokens
```

Comment:
for this particular line I'd like @ollmer to take a look

Reply:
Yep, looks correct, addresses nuances about tokens separating different chat messages.